app/models.py

```diff
@@ -5,3 +5,149 @@
 Author: huxuan <i(at)huxuan.org>
 Description: Models for app.
 """
+from app import db
+
+
+class User(db.Model):
+    __tablename__ = 'userinfo'
+
+    # new segments
+    uid = db.Column(db.Integer, primary_key=True)
+    birthday = db.Column(db.DateTime)
+    student_id = db.Column(db.String(16))
+    name = db.Column(db.String(16))
+    tel = db.Column(db.String(16))
+
+    # existing segments
+    username = db.Column(db.String(32), unique=True)
+    password = db.Column(db.String(32), nullable=False)
+    # token temporarily not included
+    # tokentime temporarily not included
+    gender = db.Column(db.Boolean, nullable=False)  # was `sex`
+    icon = db.Column(db.Text)
+    intro = db.Column(db.Text)
+    sig1 = db.Column(db.Text)
+    sig2 = db.Column(db.Text)
+    sig3 = db.Column(db.Text)
+    hobby = db.Column(db.Text)
```

Comment:
Since they do no harm, we can keep them for the time being in case of future use.